plugin/pkg/scheduler/algorithm/priorities/spreading.go

```diff
@@ -22,28 +22,42 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/plugin/pkg/scheduler/algorithm"
 )
 
-type ServiceSpread struct {
-	serviceLister algorithm.ServiceLister
+type JobSpread struct {
```

Comment:
Yeah - I thought about this, but thought that plain 'Spread' will be too general.